dispatcher/backend/src/utils/keys_exporter.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from typing import Dict, List
+
+logger = logging.getLogger(__name__)
+
+
+class KeysExporter:
```

Comment:
No docstring ; is this a migration-only tool?

Reply:
Yes, very little chance it will serve afterwards.
Should I add it as a class in the migration file instead ?

Reply:
Makes more sense to me